### PR TITLE
Add blog embed to homepage

### PR DESCRIPTION
**Pull request by rubenwardy**
image

### CODE REVIEW

**Comment by rubenwardy** [comment]
This is a bit broken on mobile, but I think the way to fix this would be in the minetest/blog repo

image

**Comment by rubenwardy** [comment]
@Calinou  please may you take a look at this?

File: _sass/_blog.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.blog-embed {
+	// position: relative;
+	text-align: center;
+
+	iframe {
+		width: 100%;
+		height: 230px;
```
Comment (appgurueu) [inline _sass/_blog.scss:7]: I don't think using `px` widths here is a good idea.

Comment (Calinou) [reply _sass/_blog.scss:7]: I prefer using `rem` but on hiDPI displays, browsers will scale `px` as needed anyway. The whole "using px units breaks hiDPI" support isn't really true due to how browsers handle px as DPI-independent pixels nowadays (by adjusting the default zoom behind the scenes).

**Pull request merged**